.travis.yml

```diff
@@ -16,6 +16,7 @@ branches:
     - release
     - dev
     - /^(.*\/)?ci-.*$/
+    - /^rel\/.*/
```

Comment:
👍 We should add a work item to track making these changes in all our repos as this would be the usual format for future releases too